src/js/components/Institutions.jsx

```diff
@@ -87,9 +87,10 @@ const renderButton = (code, institutionId, period) => {
   return <Link className="usa-button" to={`/${institutionId}/${period}`}>{buttonText}</Link>
 }
 
-const renderRefile = (code, institutionId, period) => {
-  if (code === 1) return null
-  return <Link className="usa-button usa-button-secondary usa-text-small" to={`/${institutionId}/${period}`}>Refile</Link>
+const renderRefile = (makeNewSubmission, code, institutionId, period) => {
+  if(code === 1) return null
+  return <a className="usa-button usa-button-secondary usa-text-small" onClick={()=>{
```

Comment:
Cause all we care about is the action that the click triggers, not the special behavior of the Link component.